revised/stellaswap-proposal

```diff
@@ -0,0 +1,237 @@
+
+Based on that, our target TVL would be around $20-25M - throughout the duration of the established grant duration. The failure for StellaSwap to reach a base TVL of $25M would require us to increase STELLA emissions in order to reach the $25M base. This is a sign of our commitment in ensuring that there is sufficient base liquidity to power the ecosystem.
+
+In the opposite scenario where market conditions tend to be positive and GLMR prices pick back up, we will maintain status quo and facilitate the reflection of the increased APRs that will seek to deepen ecosystem liquidity. This will be highly beneficial to the ecosystem where the flywheel effects could be enhanced, given the limitations of a relatively small ecosystem grant cap (Level 3).
+
+
+The reality is that if we don't put sufficient incentives on Moonbeam ecosystem projects, we won't attract meaningful liquidity and will continue to be a "ghost chain". StellaSwap is deeply invested in the ecosystem, and is building for the long term, but there are certain economic realities that liquidity providers have when it comes to providing liquidity in an ecosystem:
+
+* If the rewards are not sufficient in the form of APRs that justify the risk, LPs will not take the risk of using a new ecosystem, especially if bridges are involved.
+* If LPs choose to bring their liquidity to other ecosystems with higher reward APRs, there may never be enough liquidity for Moonwell to establish healthy collateral ratios for these markets because liquidations cannot be successful.
+
+**Secondary Metrics Tracking***(Updated feedback from Natalia)
+It is important for StellaSwap to also ensure the tracking of secondary metrics in a transparent manner so that we can analyze the effectiveness of the grants program. Here is the following secondary metrics that will be tracked, which is similar to prior metrics that were furnished to the Foundation during pre-Nomad times;
+
+* Avg Trading Volume (Daily)
+* Avg No. of Txns (Daily)
+* No. of Unique Users
+* Trade Fee APRs
+* Farms APR
+
+**STEPS TO IMPLEMENT**
+We've launched Pulsar mainnet at the start of the year in order to prove the power of concentrated liquidity. After close to 2 months, we've optimized Pulsar and is ready to ramp up the next phase of liquidity building. In terms of the sequential steps;
+
+- We'll begin transiting strategic farms from Standard AMM to Pulsar on the last week of March in preparation for the grants
+- Newly-transited Pulsar farms will be powered with STELLA token first
+- Once grants are received, we'll push the grants directly to the established farms on this proposal
+
+**Strategy for Farm Launches** (Updated feedback from Natalia)
+We've announced the formal plan for [farm transmission](https://stellaswap.medium.com/transition-of-strategic-farms-to-pulsar-the-plan-8d04265fac2e), which entails;
+![image|690x443](upload://wXJD2eE34ysg0ZuwzOUDDRuCQLf.jpeg)
```

Comment:
Can you add the image to the PR - otherwise this will show a blank image placeholder.